apps/list.c

```diff
@@ -1576,7 +1627,7 @@ int list_main(int argc, char **argv)
         switch (o) {
         case OPT_EOF:  /* Never hit, but suppresses warning */
         case OPT_ERR:
-opthelp:
+        opthelp:
```

Comment:
nit: don't indent this please.

Reply:
That was not my choice: I did this only to "satisfy" the style-checker tool in CI. Will revert as part of PR review update now that style:waived tag is set and further feedback is received.

Reply:
Changed back in latest commit.